### DISCUSSION TITLE
selfbuild dev version offers only two languages

### DISCUSSION THREAD

**Discussion by octaeder** [texstudio-org/texstudio#2693] [Q&A] [answered]
I build with msys2. When I open config manager and go to General I only see this:

![image](https://user-images.githubusercontent.com/102688820/202912232-f0582b6d-02c3-4373-acd1-876c49388aa7.png)

But the installed version shows many languages to choose from. It seems that both txs use the same profile, so I don't understand the difference. Is there something I can do? I assume that this has nothing to do with spellcheck and the like.

Thx

**Comment by sunderme** [comment, marked as answer]
just look where translations are placed in the portatble version, that needs to be the same for your own build.

**Reply by octaeder** [reply]
I copied the translations folder and it worked. Thx.
